OpenRA.Mods.RA2/Widgets/PowerMeterWidget.cs

```diff
@@ -0,0 +1,250 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Mods.Common.Widgets;
+using OpenRA.Widgets;
+using OpenRA.Graphics;
+using System;
+
+namespace OpenRA.Mods.RA2.Widgets.Logic
+{
+
+	public class PowerMeterWidget : Widget
+	{
+		Widget sidebar_production;
+
+		[Sync] int lastMeterCheck;
```

Comment:
Why does the UI need to sync between players? I assume this does nothing as the `ISync` interface isn't implemented.